tests/app/services/test_process_request_json.py

```diff
@@ -12,14 +11,15 @@
 def test_should_add_filter_fields_to_index_json(services_mapping):
     request = {
         "lot": "SaaS",
-        "freeOption": True,
+        "phoneSupport": True,
     }
 
     result = convert_request_json_into_index_json(services_mapping, request)
     assert_equal(result, {
```

Comment:
Can we get rid of these nose assert_equal's?

Reply:
'twould be nice, the only reason we can't purge them completely from the repo is `test_search_queries.py` still uses nose's old form of parametrized test using generators